Xamarin.Forms.Build.Tasks/TypeReferenceExtensions.cs

```diff
@@ -139,22 +139,53 @@ public static bool ImplementsInterface(this TypeReference typeRef, TypeReference
 		{
 			interfaceReference = null;
 			genericArguments = null;
-			var typeDef = typeRef.ResolveCached();
```

Comment:
bypassing the cache resolution has a huge negative impact on performances

Reply:
Not bypassing but the indent level was changed and it seemed like line deletion in diff. Now the indent change was reverted with 1b697b17ddaabe70a6e07169f626a038a29346e2.